schemas/region.schema.json

```
@@ -0,0 +1,102 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://region-registry.skedgo.com/region.schema.json",
+  "title": "Region",
+  "description": "Region specification",
+  "type": "object",
+  "properties": {
+    "code": {
+      "description": "The unique identifier for a region.",
+      "type": "string"
+    },
+    "timezone": {
+      "description": "A Java valid time zone id.",
+      "type": "string"
+    },
+    "locale": {
+      "description": "Locale information",
+      "type": "object",
+      "properties": {
+        "language": {
+          "description": "An ISO 639 alpha-2 or alpha-3 language code, or a language subtag up to 8 characters in length.",
+          "type": "string"
+        },
+        "country": {
+          "description": "An ISO 3166 alpha-2 country code or a UN M.49 numeric-3 area code.",
+          "type": "string"
+        }
+      },
+      "required": [ "language", "country" ]
+    },
+    "coverage": {
+      "description": "Coverage details.",
+      "type": "object",
+      "properties": {
+        "cities": {
+          "description": "List of cities that this region contains.",
+          "type": "array",
+          "items": { "$ref": "#/$defs/city" }
+        },
+        "polygon": {
+          "$ref": "https://geojson.org/schema/Polygon.json",
+          "additionalProperties": false
+        }
+      },
+      "required": [ "defaultLocation", "polygon" ]
+    },
+    "currency": {
+      "description": "Details of the currency used in this region.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The ISO 4217 currency code of this currency.",
+          "type": "string"
+        }
+      },
+      "required": [ "code" ]
+    }, 
```

Can now be simpliefied:

```suggestion
    "currencyCode": {
      "description": "The ISO 4217 currency code of this currency.",
      "type": "string"
    }, 
```